Example/Example/Model/ContentType.swift

```diff
@@ -18,16 +18,11 @@ enum ContentType: String, Printable {
     }
     
     var image: UIImage {
-        let image =  UIImage(named: rawValue)!
+        let image =  UIImage(named: "content_\(rawValue.lowercaseString).png")!
```

Comment:
It's not swifty) And it will increase count of the code, I will need to paste switch statement / oh seems we have different understanding of clear code)
http://www.jessesquires.com/swift-namespaced-constants/
I think better compromise will be if I add this method in enum:
private var imageName: String {
        switch self {
        case .Music:
            return "content_music.png"
        case .Films:
            return "content_films.png"
        }
}

And then let image =  UIImage(named:imageName)
Is it ok?